packages/firestore/lite/register.ts

```diff
@@ -29,6 +34,7 @@ declare module '@firebase/component' {
 }
 
 export function registerFirestore(): void {
+  setSDKVersion(SDK_VERSION);
```

Comment:
Can we differentiate Firestore lite from Firestore? I wonder if we should prefix or postfix this with `lite`.

Reply:
like `9.0.0-beta.1_lite`?

Reply:
Don't we normally put variant info in the product string and not in the version number string? It's already in the product string.

Reply:
The product string is in line 55, I think that should be enough to distinguish for platform logging, unless this is for Firestore-only backend logging? I think it might make platform logging queries a little harder to have one version not like the others.

Reply:
This version is logged to Firestore backend directly. We will have to do some data massage to join Firestore data with platform logging data.